index.html

```diff
@@ -19,27 +19,46 @@
     <!-- Enables media queries on mobile -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
+    <!-- Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;500;700;900&family=Zen+Dots&display=swap" rel="stylesheet">
+    <style>@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;500;700;900&family=Zen+Dots&display=swap');</style>
+
     <!-- CSS-->
     <link type="text/css" rel="stylesheet" href="css/main.css" />
+    <link type="text/css" rel="stylesheet" href="css/welcome.css" />
 
     <title>Regis Altare Booster Pack Simulator</title>
   </head>
 
   <body>
     <main>
-      <h1>The Regis Altare Birthday Project 2023</h1>
 
-      <nav>
-        <a class="active" href="index.html">Collection</a>
-        <a href="pages/message-board.html">Birthday Message Board</a>
-      </nav>
+      <section id="welcome-message" class="welcome-splash visually-hidden" role="region" aria-live="polite">
+        <h2>Happy Birthday Altare!</h2>
+        <h3>Let's see what the Altventurers have made for you</h3>
+        <!-- TODO: change this to an svg or img of an arrow -->
+        <button onclick="toggleWelcomeTwo()">arrow</button>
+      </section>
 
-      <section id="card-list">
-        <!--Test image serving-->
-        <img class="card-image" src="images/elysium_delivery_services.jpg" />
-        <tcg-card></tcg-card>
+      <section id="welcome-fancy-menu" class="welcome-splash visually-hidden" role="region" aria-live="polite">
```

Comment:
Remove `aria-live="polite"` here - it does not do anything because there is no text that is a direct child of it

My idea to provide alt text for the Altare illustration and announce that the page has new content in it would be like:
```
<div aria-live="polite" class="visually-hidden"> Altare leaps out, and there are 5 cards you can choose from </div>
```
added within this section
The primary purpose of this text would be to announce that the page has new content moreso than provide a thorough description of the artwork